docker-for-mac/release-notes.md

```diff
@@ -1,17 +1,55 @@
 ---
-description: Change log / release notes per Stable release
-keywords: Docker Desktop for Mac, stable, release notes
+description: Change log / release notes for Docker Desktop Mac
+keywords: Docker Desktop for Mac, release notes
 redirect_from:
 - /mackit/release-notes/
-title: Docker for Mac Stable release notes
+- /docker-for-mac/edge-release-notes/
+title: Docker for Mac release notes
 toc_min: 1
 toc_max: 2
 ---
 
-This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop Stable releases.
+This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
-For information about Edge releases, see the [Edge release notes](edge-release-notes.md). For Docker Desktop system requirements, see
-[What to know before you install](install.md#what-to-know-before-you-install).
+> **Important**
+>
+> Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
+{: .important }
+
+## Docker Desktop Community 3.0.0
+2020-12-10
+
+> [Download](ADD URL)
+
+### New
+
+- Docker Desktop updates are now much smaller as they will be applied using delta patches. For more information, see [Automatic updates](install.md#automatic-updates).
+- Use of three-digit version number for Docker Desktop releases.
+- Support for `docker compose` command for local containers (experimental) by default.
+
+  - Supports the following subcommands: `up`, `down`, `logs`, `build`, `pull`, `push`, `ls`, `ps`
+  - Supports basic volumes, bind mounts, networks, and environment variables
+
+    Let us know your feedback by creating an issue in the [compose-cli](https://github.com/docker/compose-cli/issues){: target="blank" rel="noopener" class=“”} GitHub repository.
+
+### Upgrades
```

Comment:
@mat007 @StefanScherer Should we also include Docker scan update here? If so, could you let me know the details?

Reply:
I was planning on doing a separate PR with all the release notes.
If you prefer I can send them to you and you can update this PR?
Either way works for me.

Reply:
Thanks. It would be good if you could send it to me. I'll update the entries here.